src/main/resources/sql/postgresql/schema-postgresql.sql

```
@@ -0,0 +1,348 @@
+    s3_key        VARCHAR(500),
+    article_count INT                      NOT NULL DEFAULT 0,
+    started_at    TIMESTAMP WITH TIME ZONE,
+    finished_at   TIMESTAMP WITH TIME ZONE,
+    error_message TEXT,
+
+    created_at    TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at    TIMESTAMP WITH TIME ZONE NOT NULL
+);
+
+ALTER TABLE interests
+ALTER COLUMN name TYPE VARCHAR(100) COLLATE "ko-KR-x-icu";
```

SQLFluff (4.1.0)

[error] Do not use special characters in identifiers.

(RF05)